scylla-cql/src/frame/value.rs

```diff
@@ -51,6 +51,68 @@ pub enum MaybeUnset<V> {
     Set(V),
 }
 
+/// Native CQL `varint` representation.
+///
+/// Represented as two's-complement binary in big-endian order.
```

Comment:
I think it would be useful to tell more about the purpose of this struct:

- It's a "default" implementation, independent of any external crate or crate feature, most prominently used in CqlValue
- Other, more useful types like `num_bigint::BigInt` can be used for serialization/deserialization, but they require enabling crate features.